### PR TITLE
Lesson 8

### PR DESCRIPTION
**Pull request by Ibadichan**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
-  "presets": ["env", "react"],
-  "plugins": ["transform-object-rest-spread"]
-}
```
Comment (Ibadichan) [inline .babelrc]: sdf sdf sdf sdf

Comment (Ibadichan) [inline .babelrc]: sfsdfsdf

Comment (Ibadichan) [inline .babelrc]: dsfsdf

Comment (labdoodle) [inline .babelrc]: test

Comment (Ibadichan) [inline .babelrc]: text

Comment (labdoodle) [inline .babelrc]: testing

**Pull request merged**